pkg/dns/dns.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2020 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I don't think we need the Apache license, you may want to confirm with somebody else though.